index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:wght@300;400;700&family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <nav class="navBar">
+        <img id="logo" src="./img/icon/logo.svg" alt="logo">
+        <img id="hamburguerIcon" src="./img/icon/hamburguer.svg" alt="hamburguer-icon">
+    </nav>
+    
+    <main class="mainSection">
+        <h1 class="mainTitle">I’m Damian.
+            Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+
+        <h3>LET’S CONNECT</h3>
+        <ul id="iconListContainer">
+            <li><img class="iconList" src="./img/icon/twitter.svg" alt="Twitter"></li>
+            <li><img class="iconList" src="./img/icon/linkedin.svg" alt="Linkedin"></li>
+            <li><img class="iconList" src="./img/icon/mlogo.svg" alt="mlogo"></li>
+            <li><img class="iconList" src="./img/icon/guthub.svg" alt="github"></li>
+            <li><img class="iconList" src="./img/icon/peace.svg" alt="Peace"></li>
+        </ul>
```

Comment:
- [ ] Good job so far, however, please ensure that you wrap all the contents of your main with `section` tag according to the project requirement.

Reply:
OK! Done.
![image](https://github.com/DamianDib9/personal-portfolio/assets/121461459/7f7815a7-528d-48d6-a144-d2fe00f9eaa7)